### PR TITLE
planning: initialized stages in park_and_go scenario

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/planning/proto/planning_config.proto [hunk 1]
```diff
@@ -219,6 +219,11 @@ message ScenarioConfig {
     // valet parking scenario
     VALET_PARKING_APPROACHING_PARKING_SPOT = 510;
     VALET_PARKING_PARKING = 511;
+
+    // hailing scenario
+    HAILING_TRY = 601;
+    HAILING_CRUISE = 602;
```
Comment (Capri2014) [inline modules/planning/proto/planning_config.proto]: What is the second stage designed for ?

File: modules/planning/conf/scenario/park_and_go_config.pb.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+scenario_type: PARK_AND_GO
+park_and_go_config:
+{
+}
+
+stage_type: PARK_AND_GO_TRY
+stage_type: PARK_AND_GO_CRUISE
+stage_type: PARK_AND_GO_ADJUST
+
```
Comment (jmtao) [inline modules/planning/conf/scenario/park_and_go_config.pb.txt]: switch stage #2 and #3. 
try (yes) -> cruise
try (fail) -> adjust -> cruise

and probably change "try" to "check"?

File: modules/planning/common/planning_gflags.h [hunk 3]
```diff
@@ -29,7 +29,7 @@ DECLARE_string(scenario_lane_follow_config_file);
 DECLARE_string(scenario_narrow_street_u_turn_config_file);
 DECLARE_string(scenario_pull_over_config_file);
 DECLARE_string(scenario_stop_sign_unprotected_config_file);
-DECLARE_string(scenario_hailing_config_file);
+DECLARE_string(scenario_park_and_go_config_file);
```
Comment (jmtao) [inline modules/planning/common/planning_gflags.h]: add a gflag for this feature and set it to false for now.

File: modules/planning/scenarios/park_and_go/park_and_go_scenario_test.cc [hunk 4]
```diff
@@ -18,7 +18,7 @@
  * @file
  **/
```
Comment (jmtao) [inline modules/planning/scenarios/park_and_go/park_and_go_scenario_test.cc]: folder name?

File: modules/planning/scenarios/park_and_go/stage_try.cc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+namespace apollo {
+namespace planning {
+namespace scenario {
+namespace park_and_go {
+
+using common::TrajectoryPoint;
+
+Stage::StageStatus ParkAndGoStageTry::Process(
+    const TrajectoryPoint& planning_init_point, Frame* frame) {
+  return FinishStage(true);
```
Comment (jmtao) [inline modules/planning/scenarios/park_and_go/stage_try.cc]: add comment line of TODO

**Pull request merged**